content/docs/integrating-qri/r-studio.md

```diff
@@ -0,0 +1,201 @@
+
+glimpse(turnstile_counts_2020)
+
+Rows: 113,033
+Columns: 12
+$ stop_name      <chr> "Astoria - Ditmars Blvd", "Astoria - Ditmars Blvd", "Astoria - Ditmars Blvd", …
+$ daytime_routes <chr> "N W", "N W", "N W", "N W", "N W", "N W", "N W", "N W", "N W", "N W", "N W", "…
+$ division       <chr> "BMT", "BMT", "BMT", "BMT", "BMT", "BMT", "BMT", "BMT", "BMT", "BMT", "BMT", "…
+$ line           <chr> "Astoria", "Astoria", "Astoria", "Astoria", "Astoria", "Astoria", "Astoria", "…
+$ borough        <chr> "Q", "Q", "Q", "Q", "Q", "Q", "Q", "Q", "Q", "Q", "Q", "Q", "Q", "Q", "Q", "Q"…
+$ structure      <chr> "Elevated", "Elevated", "Elevated", "Elevated", "Elevated", "Elevated", "Eleva…
+$ gtfs_longitude <dbl> -73.91203, -73.91203, -73.91203, -73.91203, -73.91203, -73.91203, -73.91203, -…
+$ gtfs_latitude  <dbl> 40.77504, 40.77504, 40.77504, 40.77504, 40.77504, 40.77504, 40.77504, 40.77504…
+$ complex_id     <chr> "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"…
+$ date           <chr> "2020-01-01", "2020-01-02", "2020-01-03", "2020-01-04", "2020-01-05", "2020-01…
+$ entries        <int> 5812, 15506, 15959, 9265, 7191, 16810, 17436, 17575, 17554, 17529, 10304, 7418…
+$ exits          <int> 6659, 14307, 14500, 9161, 7743, 14919, 14765, 15340, 15112, 15250, 9816, 8157,…
+```
+
+There it is, and all the columns got their types from the qri dataset!
+
+## Get your R on
+
+Now that the dataframe is imported, we can do some filtering and make a quick chart.
+
+```{r}
+# Let's make a dataframe just for one station complex, Atlantic Ave/Barclays Center
+barclays_2020 <- filter(turnstile_counts_2020, complex_id == '617')
+
+# neat.  How many entries were there at Atlantic Ave Barclays Center
+# the week of August 30 - September 5?
+barclays_aug30_week <- filter(
+  barclays_2020,
+  date >= '2020-08-30',
+  date <= '2020-09-05'
+)
+
+# let's make a simple chart to visualize the entries and exits over the course of the week
+fig <- plot_ly(
+  barclays_aug30_week,
+  x = ~date,
+  y = ~entries,
+  type = 'scatter',
+  mode = 'lines',
+  name = 'entries',
+  line = list(color = 'rgb(205, 12, 24)', width = 4)
+) %>%
+
+  add_trace(y = ~exits, name = 'exits', line = list(color = 'rgb(22, 96, 167)', width = 4)) %>%
+
+  layout(
+    title = "Daily Entries and Exits - Atlantic Ave/Barclays Center Station Complex",
+    xaxis = list(
+      title = "Date",
+      type = 'date',
+      tickformat = '%d %B (%a)'
+    ),
+    yaxis = list(
+      title = "Count",
+      range = c(0, 15000)
+    ))
+
+fig
+
+```
+
+<ImageWithCaption
+  src='/img/docs/integrating-qri/rplot.png'
+  caption="A simple plotly chart"
+/>
+
+
+## Saving an R dataframe to a new qri dataset
+
+After filtering, mutating, and summarizing, you may have a new dataframe to share with the world.  You could export a CSV, but where would you put it?  Github maybe, but qri is a better option for publishing versioned datasets.  
```

Comment:
I'd drop the last sentence. Mainly because we don't know what the reader's first thought for publishing is. It might be to publish as an [R data package](https://www.erikhoward.net/blog/how-to-create-an-r-data-package/)

Reply:
👍